app/scripts/lib/metrics.js

```diff
@@ -98,6 +98,7 @@ define([
     this._marketingImpressions = {};
 
     this._able = options.able;
+    this._uuid = options.uuid;
```

Comment:
Something about putting uuid on metrics doesn't seem quite write, esp as a private member variable which is then [accessed by external consumers](https://github.com/mozilla/fxa-content-server/pull/2456/files#diff-a74f50cdf368ca0e8f24665d7488f14dR205).

This seems like data that belongs on the [user model](https://github.com/mozilla/fxa-content-server/blob/master/app/scripts/models/user.js) instead, since the intent of the uuid is to give the user a unique identifier that can be used for both metrics and for choosing AB test buckets.